src/main/java/at/hannibal2/skyhanni/features/misc/update/CustomGithubReleaseUpdateSource.kt

```diff
@@ -11,6 +12,11 @@ import moe.nea.libautoupdate.UpdateData
  */
 class CustomGithubReleaseUpdateSource(owner: String, repository: String) : GithubReleaseUpdateSource(owner, repository) {
 
+    override fun findLatestRelease(validReleases: Iterable<GithubRelease>): UpdateData {
+        return validReleases.asSequence().maxBy { ModVersion.fromString(it.tagName) }.let { findAsset(it) }
+            ?: throw IllegalStateException("No valid release found")
```

Comment:
im pretty sure traditionally this just returns null, but that is just fine. it will practically never happen, and even if it did, it would still just be caught.